### PR TITLE
initial hw 3 tests upload

### PR DESCRIPTION
**Pull request by sshamilova**

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    });
+
+});
+
+//Найти сумму четных чисел в диапазоне от 1 до 99.
+describe('getEvenSum', () => {
+    it('should return 2500', () => {
+        const expected = 2500;
+
+        const actual = getEvenSum( );
```
Comment (Daymon4) [inline test/index.test.js]: no space between brackets

File: test/index.test.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+    it('should return true (13)', () => {
+        const number = 13;
+        const expected = false;
+
+        const actual = getIsPrime(number);
+
+        assert.deepEqual(actual, expected);
+    });
+
```
Comment (Daymon4) [inline test/index.test.js]: add more cases for odd but not prime

File: test/index.test.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    it('should return A (95)', () => {
+        const rate = 95;
+        const expected = 'A';
+
+        const actual = getMark(rate);
+
+        assert.deepEqual(actual, expected);
+    });
+
```
[the remark this answers is not in the document]
Comment (sshamilova) [reply test/index.test.js]: if (rate >= 90 && rate <= 100) we have this condition, and have tested it already, for the rest comments I am ok

File: test/index.test.js [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+        const first = -1;
+        const second = 2;
+        const third = 3;
+        const expected = 7;
+
+        const actual = getMaxOfProductsSum(first, second, third);
+
+        assert.deepEqual(actual, expected);
+    });
+
```
[the remark this answers is not in the document]
Comment (sshamilova) [reply test/index.test.js]: Really do not have any empty row, maybe it is some visual tricks of GitHub

**Pull request closed without merging**